lib/metadata/util/md5deep.rb

```diff
@@ -226,10 +226,18 @@ def calculate_digest(fileName)
       # Create hash for requested digests
       digest = create_digest_hash
 
-      fileName.seek(0, IO::SEEK_SET)
-      # Loop over each digest and add the file contents
-      while buf = fileName.read(10240000)
-        digest.each_pair { |_k, v| v << buf }
+      begin
+        fileName.seek(0, IO::SEEK_SET)
+        # Loop over each digest and add the file contents
+        while (buf = fileName.read(10_240_000))
+          digest.each_pair { |_k, v| v << buf }
+        end
+      rescue NoMethodError => method_err
```

Comment:
Why do we get NoMehtodError? Wouldn't that mean there's a bug somewhere else in the code?

Reply:
Well, yes.  And I did catch that bug with my change in fs/ntfs/data_run.rb.  I thought perhaps catching it even though I haven't seen it could address other issues that bad data on the disk could create.  I'd prefer to just catch the Exception class but our CodeClimate settings don't allow that.  If you prefer I can just leave in the "Rescue => err" clause, or I am open to other suggestions.

Reply:
@roliveri removed the redundant rescue of NoMethodError as requested.  This should be ok to go with your approval.